### DISCUSSION TITLE
Tracking progress in the "Projects" area

### DISCUSSION THREAD

**Discussion by QuintillusCFC** [C7-Game/Prototype#231] [General]
One of the features GitHub gives us is project boards; the Carthage one is at https://github.com/C7-Game/Prototype/projects/3 .  I've noticed that a few items have been added recently that aren't part of it ([link](https://github.com/C7-Game/Prototype/pull/227), [link](https://github.com/C7-Game/Prototype/pull/229)).  So, I thought it made sense to have a topic about why the effort has been put into creating the project board.

The short version is, there are a several benefits, not just one.  These include:

 - Easy to find a task that helps contribute towards the goals of the next release (which are based on community feedback).  This has been going well.
 - Easy to see if someone else is already working on something, so we don't have two people working on the same thing.  This is a particular concern since in Babylon, there were some large tasks done that weren't visible due to being done in branches that weren't tracked on the project board, or posted about much at CFC or other venues.
 - Getting credit for your work in communications.  When releases are made, that board is the go-to place for checking what is done, since there may be multiple PRs for individual tasks.
 - Figuring out how far along we are towards our top-level milestones ("Babylon", "Carthage", etc.).  This in turn helps us have a more realistic view of how far we have to go.

It will be the case that not everything is thought of up front; there will be Carthage-worthy tasks that aren't on that board yet.  What I'd ask is that if you are adding something, create an issue for it, put it on the "Carthage" project, optionally assign a [milestone](https://github.com/C7-Game/Prototype/milestones) label if appropriate, and link the PR.  Or at least, link the PR directly to the "Carthage" project:

![image](https://user-images.githubusercontent.com/5287977/162817417-a4839d09-42e9-43be-8ac1-42e972705816.png)

This will help keep the project organized and help communicate progress to non-technical interested parties, and will become more important if the project continues to grow.  Basically, it's another way, like the recent editor coding config and logging additions, that shows C7 is a maturing project, and an approachable one for new developers.

**Comment by WildWeazel** [comment]
Good post! I'll also add that we should get in the habit of checking, when there's activity on an issue or PR, that it has the appropriate labels, project, etc in the sidebar.
